hazelcast/src/main/java/com/hazelcast/core/IndeterminateOperationState.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.core;
+
+/**
+ * A marker interface for exceptions to denote that execution status of an invocation is not known
+ */
+public interface IndeterminateOperationState {
```

Comment:
Minor: it was a bit confusing that `IndeterminateOperationStateException` doesn't implement `IndeterminateOperationState`